src/spring/azext_spring/_params.py

```diff
@@ -861,6 +861,10 @@ def prepare_logs_argument(c):
                    help='Sensitive properties for environment variables. Once put, it will be encrypted and not returned.'
                         'Format "key[=value]" and separated by space.')
         c.argument('allowed_origins', arg_group='Cross-origin Resource Sharing (CORS)', help="Comma-separated list of allowed origins to make cross-site requests. The special value `*` allows all domains.")
```

Comment:
It is recommended to keep the naming of parameter `allowed_origins` consistent with parameter `allowed_origin_patterns`

Reply:
As talked offline, we will keep the name consistent with other parameters first, which is referenced from VMware's product Spring Cloud Gateway. Later will consider to deprecate those parameters together.